### ISSUE TITLE
Eliminate the eval command

### ISSUE THREAD

**Issue by wsnoble** [Noble-Lab/casanovo#355]
The eval command is just like the sequence command except that it requires annotated spectra.  We should get rid of it, and then have the program automatically do evaluations if the user provides annotated spectra.  If no validation set is provided, the evaluation should be done on the training set.  Results can be printed to the log file, as is done currently.

**Comment by bittremieux** [comment]
> have the program automatically do evaluations if the user provides annotated spectra
This is a bit tricky, as you need to inspect the file first to check whether there are peptide sequences included in order to know how to instantiate the data loader.

Easier would be the alternative option that adds an `--evaluate` flag.

**Comment by wsnoble** [comment]
Yes, --evaluate is fine with me.

**Comment by Lilferrit** [comment]
Do we still want to output the sequence results file if the evaluate flag is set?

**Comment by wsnoble** [comment]
Yes, we should output sequencing results if the flag is set.
